### PR TITLE
Internet Explorer fixes

### PR DESCRIPTION
**Pull request by captn3m0**

### CODE REVIEW

**Comment by ankitstarski** [comment]
:+1: LGTM

**Comment by ankitstarski** [comment]
@captn3m0 you might want to remove `onclick` from this line [1], otherwise `openCheckout()` will fire twice.

 [1] https://github.com/razorpay/razorpay-woocommerce/blob/303644c84fc652d13329e57186f667a4a660ccea/razorpay-payments.php#L179

File: razorpay-payments.php [hunk 1]
```diff
@@ -180,9 +180,13 @@ public function generate_razorpay_form($order_id)
     <button id="btn-razorpay-cancel" onclick="document.razorpayform.submit()">Cancel</button>
 </p>
 <script>
-    var setDisabled = function(id, state = true) {
```
Comment (ankitstarski) [inline razorpay-payments.php]: default state is `true` here and `false` after patch

**Comment by captn3m0** [comment]
@ankitstarski The onclick was removed. change got stashed accidentally.

File: razorpay-payments.php (hunk 1 above)
Comment (captn3m0) [inline razorpay-payments.php]: Oops. Will fix.

File: razorpay-payments.php [hunk 2]
```diff
@@ -215,8 +219,20 @@ function openCheckout() {
       // Disable the pay button
       setDisabled('btn-razorpay');
```
Comment (ankitstarski) [inline razorpay-payments.php]: This may enable the `pay-razorpay` button

**Pull request merged**